addons/goggles/CfgEventHandlers.hpp

```diff
@@ -14,15 +14,15 @@ class Extended_PostInit_EventHandlers {
 class Extended_Killed_EventHandlers {
     class CAManBase {
         class ADDON {
-            killed = QUOTE(_this call FUNC(handleKilled));
+            clientKilled = QUOTE(_this call FUNC(handleKilled));
         };
     };
 };
 
-class Extended_Fired_EventHandlers {
+class Extended_FiredBIS_EventHandlers {
     class CAManBase {
         class ADDON {
-            fired = QUOTE(if (local (_this select 0)) then {_this call FUNC(handleFired)});
+            clientFiredBIS = QUOTE(if (local (_this select 0)) then {_this call FUNC(handleFired)});
```

Comment:
clientX XEH is executed on every machine where `!isDedicated` is true.

A local prefix would be cool too (localFiredBIS, localGetIn, localEngine), but it doesn't really make sense on some events that are always limited to local machines. (respawn, killed, weaponRested)